server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -224,33 +224,46 @@ export class BasicRestWrapper extends RestWrapper {
 
 						this.request<T>(retryConfig, statusCode, false).then(resolve).catch(reject);
 					} else {
+						const errorSourceMessage = `[${error?.config?.method ?? ""}] request to [${
+							error?.config?.baseURL ?? options.baseURL ?? ""
+						}] failed with [${error.response?.status}] status code`;
 						// From https://axios-http.com/docs/handling_errors
 						if (error?.response) {
 							// The request was made and the server responded with a status code
 							// that falls out of the range of 2xx
-							reject(
-								createFluidServiceNetworkError(
-									error?.response?.status,
-									error?.response?.data,
-								),
-							);
+							if (typeof error?.response?.data === "string") {
+								reject(
+									createFluidServiceNetworkError(error?.response?.status, {
+										message: error?.response?.data,
```

Comment:
Super nit, will `error.response.data` has cc content? But it was look like this already